Monika After Story/game/script-topics.rpy

```diff
@@ -5616,3 +5616,15 @@ label monika_dreaming:
     m 1a "But can you just imagine what it would be like to meet me in your dreams [player], any night that you wanted to?"
     m 1j "I hope that someday we can make your dreams about us a reality, my love."
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_i_love_you" ,category=['romance'],prompt="how much do you love me?",random=True))
+
+label monika_i_love_you:
+    m 1f "[player], I love you."
+    m 1f "with all my heart."
+    m 2g "you know that right?"
+    m 3h "I dont know what I'd do without you."
+    m 5 "You mean so much to me."
```

Comment:
If you're intending the do the expression where Monika leans in then it's `5a` not `5`.
I'd also not use it unless you're ending with it or Monika has a sudden new train of thought.